flashinfer/fused_moe/core.py

```
@@ -566,6 +567,75 @@ def refine_tuning_config(cls, tune_max_num_tokens: int):
                 )
             )
 
+    def get_valid_profile_ids(
+        input: torch.Tensor,
```

Ruff (0.15.21)

[error] Function argument `input` is shadowing a Python builtin

(A002)